CONTRIBUTING.md

```diff
@@ -3,7 +3,6 @@
 - [How to Contribute](#how-to-contribute)
   - [Contributing via Pull Requests](#contributing-via-pull-requests)
     - [Adding support for a new language](#adding-support-for-a-new-language)
-      - [Ascii logo](#ascii-logo)
```

Comment:
TBH, now that GitHub has its own table of contents for Markdown files, we can probably remove this table of contents altogether 🙂

Reply:
You're right, I didn't even know about this (https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/) 😅